Documentation/v2/docker_guide.md

```diff
@@ -16,7 +16,7 @@ This will run the latest release version of etcd. You can specify version if nee
 
 ```
 docker run -d -v /usr/share/ca-certificates/:/etc/ssl/certs -p 4001:4001 -p 2380:2380 -p 2379:2379 \
- --name etcd quay.io/coreos/etcd \
+ --name etcd quay.io/coreos/etcd /usr/local/bin/etcd \
```

Comment:
I prefer absolute for docker run,  but can change it to relative if that is preferred.